spec/factories/school_targets.rb

```diff
@@ -10,5 +10,30 @@
     electricity_progress { {} }
     gas_progress { {} }
     storage_heaters_progress { {} }
+
+    trait :with_progress_report do
+      transient do
+        fuel_type { :electricity }
+      end
+
+      after(:build) do |target, evaluator|
+        report = {
+          'fuel_type': 'electricity',
+          'months': ['2024-01-01'],
```

Comment:
don't think it makes a difference these are quoted

Reply:
I was leaving them as string as that's how it'll come out of the database. There's code in the school target model to symbolize them and reformat some values when its converted into an object. So was preserving the database version.

Reply:
then it should be `'months' => ['2024-01-01']`?